src/routes/index.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import userRoutes from './user.route';
+
+const routes = Router();
+
+routes.get('/', (req, res) => {
+  res.status(200).json({
+    status: 200,
```

Comment:
remove this status from the response body. It's already in the response status

Reply:
Okay, I will remove it.